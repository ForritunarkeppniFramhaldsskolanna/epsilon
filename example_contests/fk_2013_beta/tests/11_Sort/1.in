4
2 3 1 4
1 2 3 4
10 -8 4 7 7 8 9
-8 4 7 7 9 8 10
10 11 12
10 11 12
1 2 3 4
5 6 7 8
