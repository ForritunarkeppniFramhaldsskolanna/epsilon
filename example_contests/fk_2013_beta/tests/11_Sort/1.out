Accepted
Wrong Answer
Accepted
Wrong Answer
